docs/src/plotting_utils.jl

```diff
@@ -94,9 +95,8 @@ function convergence_order(dts, errs, confidence)
 end
 
 function make_saving_callback(cb, u, t, integrator)
-    DECB = CTS.DiffEqCallbacks
     savevalType = typeof(cb(u, t, integrator))
-    return DECB.SavingCallback(cb, DECB.SavedValues(typeof(t), savevalType))
+    return DiffEqCallbacks.SavingCallback(cb, DiffEqCallbacks.SavedValues(typeof(t), savevalType))
```

Comment:
Can/should we just use the CTS `SavedValues` here? Is the `SavingCallback` definition too large to port over too?

Reply:
Yes, we can use `CTS.SavedValues`, but we would still need `SavingCallback`. `SavingCallback` depends on a bunch of SciML stuff, so it is not completely trivial to port over. Instread, I tried using `NonInterpolatingSavingCallback` instead of `SavingCallback` but couldn't get it to work (the interfaces are slightly different).

I am sure we could get rid of this here, but since this is used only in the docs, I would rather not invest more time on this at this point.